packages/design-system/vite.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from "vite";
+import { resolve } from "node:path";
+
+export default defineConfig({
+  build: {
+    outDir: "dist",
+    cssMinify: false,
+    minify: true,
+    sourcemap: false,
```

Comment:
I believe minification is enabled by default, I think I needed to set this in mine because we used rolldown options to prevent certain optimization/minification tasks occurring as it caused some issues.